package.json

```diff
@@ -1,9 +1,9 @@
 {
-  "name": "node-nuodb",
-  "author": "NuoDB, Inc.",
+  "name": "nuodb",
+  "author": "Dassault Systèmes SE",
   "description": "The official NuoDB driver for Node.js. Provides a high-level SQL API on top of the NuoDB Node.js Addon.",
```

Comment:
Since the author is now Dassault Systemes SE, I suggesting adding the word "database" after "NuoDB":
"The official NuoDB database driver....."

Reply:
Applied suggestion